tests/test_clearml_logger.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+
+def setUpModule():
+    unittest.defaultTestLoader.parallelism = 1
+
+
+class TestClearMLLogger(unittest.TestCase):
+    def test_task_init(self):
+        from clearml import Task
```

Comment:
Does this mean it fails if `clearml` is not installed? Could you add a skip condition if it is not installed?
It should be easy to add the skip condition. Check [this PR](https://github.com/facebookresearch/ParlAI/commit/33a51a9bf9268e9cc4e3dd5d9d0b714bef486c08) to see how.

Reply:
Skip condition added. Thank you. Please, review the PR. @mojtaba-komeili